watchtower/server/interface.go

```diff
@@ -0,0 +1,66 @@
+package server
+
+import (
+	"io"
+	"net"
+	"time"
+
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/lightningnetwork/lnd/watchtower/wtdb"
+)
+
+// Interface represents a simple, listen-only service that accepts watchtower
+// clients, and provides responses to their requests.
+type Interface interface {
```

Comment:
Bit of a naming collision here ;) 

Although it's not so bad when using the full import: `server.Interface`. Although in isolation, the server package name may conflict a bit with local variables we have in `lnd` or w/e else since it's so plain.

Reply:
yeah was thinking about renaming it to something better, any suggetions??

Reply:
`TowerInterface`? Lol, not blocking atm, so we can pick a better name in the future.